### ISSUE TITLE
[T2 - P1] Duda backtranking de Tiles iniciales

### ISSUE THREAD

**Issue by aplaza2** [IIC2133-PUC/2023-2#133]
Hola, me surgió la duda de si al momento de hacer backtracking para generar el mapa sucediera que llegamos a uno de los Tiles agregados inicialmente. ¿El Tile puede ser reemplazado por otro Tile o debe quedarse en el mapa final en su posición que fue colocado inicialmente?

**Comment by benjavicente** [comment]
No, los tiles iniciales son fijos (añadiendo restricciones iniciales). No deberás tratar de cambiarlos.
